app/controllers/users_controller.rb

```diff
@@ -1,5 +1,7 @@
 class UsersController < ApplicationController
   def index; end
 
-  def show; end
+  def show
+    @user_id = params[:id]
+  end
```

Comment:
- [ ] Optional suggestion
This works perfectly, however, I would recommend this way `@user_id = User.find(params[:id])`

Reply:
Thank you for this helpful recommendation.